src/modules/ekf2/ekf2_main.cpp

```diff
@@ -1250,6 +1259,10 @@ void Ekf2::run()
 		if (vehicle_land_detected_updated) {
 			if (orb_copy(ORB_ID(vehicle_land_detected), _vehicle_land_detected_sub, &vehicle_land_detected) == PX4_OK) {
 				_ekf.set_in_air_status(!vehicle_land_detected.landed);
+
+				if (_gnd_effect_deadzone.get() > 0.0f) {
```

Comment:
@EliaTarasov We need to combine this with the distance to ground check. Basically if we have distance to ground we use that otherwise we use the flag from the landing detector.